### ISSUE TITLE
Still alive? Need capability to dump graph/topology of connected peers

### ISSUE THREAD

**Issue by adhisimon** [Kuebler-IT/MeshVPN#11]
PeerVPN is very simple to use, much simpler than tinc. But one thing I am missing from peervpn, feature to dump graph of connected peers.

PeerVPN is stale, I hope MeshVPN can implement it. But I see it is going to stale too.

**Comment by satter** [comment]
There was a debug feature in the original peervpn project:

```
you can set the "enableconsole yes" option in your config file. This enables a debug console, where you can press "a" to see a list of connected peers.

In the "Address" field, you will find different prefixes:

01040100 for IPv4 peers
01060100 for IPv6 peers
00000000 for indirect peers
```

**Comment by bkuebler** [comment]
Sorry for this much late response, i had less time for this project the last years. but in 2022 i will go forward with it.

**Comment by darkpixel** [comment]
It's 2023. ;)

**Comment by risharde** [comment]
Yes we definitely need this to continue, this is the easiest mesh vpn to get setup!

**Comment by bkuebler** [comment]
Okay will try it this year. Any wishes how it should look like?

I also appreciate PRs with possible ideas.

I started a whole rebuild last year in go for faster development. But maybe I will switch back to add those features here.
